installer/mac/README.md

```diff
@@ -9,12 +9,12 @@
 - The command below will run a script to check for existing installations of prerequisites. If a compatible version is installed, it will say which steps can be skipped below.
 
     ```bash
-    curl -s "https://raw.githubusercontent.com/jonescompneurolab/hnn/master/installer/mac/check-pre.sh" | bash
+    curl -s "https://raw.githubusercontent.com/blakecaldwell/hnn/integration_docs/installer/mac/check-pre.sh" | bash
```

Comment:
why is this pointing to your fork?

Reply:
Because you all are using this to test HNN GUI and `check-pre.sh` on master is for the old version. :)

Reply:
Also, thanks for the detailed look! You must have been looking closely